modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,41 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.3
+
+This release of {SMProductName} makes {SMProductName} operator to ARM based clusters available as a Technology Preview feature, adds the `envoyExtAuthzGrpc` field to configure an external authorization provider using the gRPC API, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.10 and later versions.
```

Comment:
```suggestion
* The {SMProductName} Operator is now available on ARM based clusters as a Technology Preview feature.
* The `envoyExtAuthzGrpc` field has been added, which is used to configure an external authorization provider using the gRPC API.
* Common Vulnerabilities and Exposures (CVEs) have been addressed.
* This release is supported on {product-title} 4.10 and newer versions.
```

Suggestion to make this a bullet list so that it's more readible, also some text editing.